core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -391,7 +391,19 @@ else
     MY_START_PORT=`expr 24000 + $MY_START_PORT '*' 200`
   elif [ $MY_START_PORT == "rand" ]; then
     # pick a random number between 12000 and 24000 that is divisible by 200
-    MY_START_PORT=`expr $RANDOM '%' 60 '*' 200 + 12000`
+    lv_number=0   #initialize
```

Comment:
Thanks, from what you told me off-line it looks that this may be a merge conflict from another branch and changing it back is the right conflict resolution.